packages/qunit-dom/package.json

```diff
@@ -50,7 +45,19 @@
     "testPathIgnorePatterns": [
       "/node_modules/",
       "/dist/"
-    ]
+    ],
+   "extensionsToTreatAsEsm": [".ts"],
+    "moduleNameMapper": {
+      "^(\\.{1,2}/.*)\\.js$": "$1"
+    },
```

Comment:
What does it do?

Reply:
no clue, jest is too hard and this didn't wor -- that's why in https://github.com/mainmatter/qunit-dom/pull/2066 I switched everything to vitest